platform/server-config/pom.xml

```diff
@@ -76,12 +76,12 @@
                 <configuration>
                     <instructions>
                         <Export-Package>
+                            org.motechproject.config.domain;version=${project.version},
+                            org.motechproject.config.monitor;version=${project.version},
+                            org.motechproject.config.service;version=${project.version},
                             org.motechproject.server.config;version=${project.version},
```

Comment:
I feel this is a good occasion to get rid of the server.config and config packages - from what I recall this is purely legacy